<launch>
  <node name="calculate_workspace" pkg="simple_reachability" type="calculate_workspace" output="screen">
    <rosparam command="load" file="$(find simple_reachability)/config/calculation.yaml"/>
  </node>

</launch>
